fluent-bundle/src/types/number.rs

```diff
@@ -8,6 +8,28 @@ use intl_pluralrules::operands::PluralOperands;
 use crate::args::FluentArgs;
 use crate::types::FluentValue;
 
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum FluentNumberType {
+    Cardinal,
+    Ordinal,
+}
+
+impl std::default::Default for FluentNumberType {
```

Comment:
This can be:

```
#[derive(Debug, Copy, Clone, Default, Hash, PartialEq, Eq)]
pub enum FluentNumberType {
    #[default]
    Cardinal,
    Ordinal,
}
```

And then you don't need the `impl std::default::Default`

I've made this change for the other enums here in #356, which will end up probably conflicting with this PR.